sdk/core/Azure.Core/tests/TestFramework/RecordTransport.cs

```diff
@@ -65,7 +56,16 @@ public override Request CreateRequest()
             return request;
         }
 
-        public RecordEntry CreateEntry(Request request, Response response)
+        private void Record(HttpMessage message)
```

Comment:
Is there a functional change here?

Reply:
No, only change in method order (to have all private methods together).